packages/overlay/package.json

```diff
@@ -30,10 +30,7 @@
     "module": "./dist/prod.js",
     "types": "./dist/types/index.d.ts",
     "exports": {
-        "browser": {
-            "development": "./dist/dev.js",
-            "default": "./dist/prod.js"
-        },
```

Comment:
These were redundant, the having the condition present shouldn't prevent finding the less-conditional exports.

Reply:
FWIW it seems to work fine for me after having removed them (on vite). But happy to revert if you think it'll cause issues.

Reply:
ok let's try with it removed

Reply:
actually neither were correct 😛
here are the updated conditions: https://github.com/thetarnav/solid-devtools/blob/main/packages/overlay/package.json#L32

Reply:
That seems.. very excessive. Dare I ask where this failed before?

Reply:
the dev runtime was getting imported during SSR on solid start
so "browser" -> "development" makes sure it gets loaded only on the client
but these consitions are usually very fragile
any electron, deno or other runtime might just randomly import the wrong thing
so I wouldn't be supprised it this one too would have to be changed

Reply:
Ah, fun.. :/